cellprofiler/modules/savecroppedobjects.py

```diff
@@ -76,12 +86,21 @@ def run(self, workspace):
         for label in unique_labels:
             mask = labels == label
 
-            filename = os.path.join(
-                directory,
-                "{}_{:04d}_{}.tiff".format(self.objects_name.value, label, int(time.time()))
-            )
+            if self.file_format.value == "png":
+                filename = os.path.join(
+                    directory,
+                    "{}_{}.png".format(self.objects_name.value, label)
+                    )
+
+                skimage.io.imsave(filename, skimage.img_as_ubyte(mask))
+
+            elif self.file_format.value == "tiff":
+                filename = os.path.join(
+                    directory,
+                    "{}_{}.tiff".format(self.objects_name.value, label)
+                    )
 
-            skimage.io.imsave(filename, skimage.img_as_ubyte(mask))
+                skimage.io.imsave(filename, skimage.img_as_ubyte(mask), compress=6)
```

Comment:
Would it make sense to allow users to specify this compression level if the `tiff` option was selected?

Reply:
@karhohs asked me about that and I said it was probably worth avoiding since, in my experience, TIFF libraries have varying levels of support for compression. @AetherUnbound what do you think?

Reply:
That's fair. Also, I can't imagine a time myself where I'd want _less_ compression, if anything I'd want more. 6 is good :)

Reply:
6 was more or less an arbitrary choice. I tried a few different levels, but the results were all similar. I this is likely due to the binary and sparse nature of the masks. A little compression goes a long ways.